profiles/license_groups

```diff
@@ -71,7 +71,7 @@ BINARY-REDISTRIBUTABLE @FREE bh-luxi Broadcom Dina intel-ucode ipw2100-fw ipw220
 
 # License agreements that try to take away your rights. These are more
 # restrictive than "all-rights-reserved" or require explicit approval.
-EULA Adaptec-EULA AdobeFlash-11.x ArxFatalis-EULA-JoWooD baudline BCS bf1942-lnxded CAPYBARA-EULA Coherent-Graphics CROSSOVER-2 CYANA DOOM3 dotnet-eula ETQW FAH-EULA-2014 FraunhoferFDK GameFront Gameplay-Group-EULA genymotion GIMPS GOG-EULA google-chrome Google-TOS Intel-SDP Introversion LastPass LOKI-EULA LRCTF MakeMKV-EULA MARBLEBLAST Mendeley-terms MTA-0.5 NVIDIA-CODEC-SDK NVIDIA-CUDA Nero-AAC-EULA Nero-EULA-US OPERA-12 OPERA-2014 Oracle-BCLA-JavaSE PAPERS-PLEASE POMPOM postal2 Primate-Plunge PUEL Q3AEULA Q3AEULA-20000111 QUAKE4 Quartus-prime-megacore RAR RTCW RTCW-ETEULA RuneScape-EULA SJ-Labs spin-commercial spin-educational StarUML-EULA TeamViewer THINKTANKS UPEK-SDK-EULA ut2003 ut2003-demo Vivaldi worklog-assistant Xerox yangcli-pro zi-labone
+EULA Adaptec-EULA AdobeFlash-11.x ArxFatalis-EULA-JoWooD baudline BCS bf1942-lnxded CAPYBARA-EULA Coherent-Graphics CROSSOVER-2 CYANA DOOM3 dotnet-eula ETQW FAH-EULA-2014 FraunhoferFDK GameFront Gameplay-Group-EULA genymotion GIMPS GOG-EULA google-chrome Google-TOS Intel-SDP Introversion LastPass LOKI-EULA LRCTF MakeMKV-EULA MARBLEBLAST Mendeley-terms MTA-0.5 NVIDIA-CODEC-SDK NVIDIA-CUDA Nero-AAC-EULA Nero-EULA-US OPERA-12 OPERA-2014 Oracle-BCLA-JavaSE PAPERS-PLEASE POMPOM postal2 Primate-Plunge PUEL Q3AEULA Q3AEULA-20000111 QUAKE4 Quartus-prime-megacore RAR RTCW RTCW-ETEULA RuneScape-EULA SJ-Labs spin-commercial spin-educational StarUML-EULA teamspeak3 TeamViewer THINKTANKS UPEK-SDK-EULA ut2003 ut2003-demo Vivaldi worklog-assistant Xerox yangcli-pro zi-labone
```

Comment:
This doesn't seem to apply, please rebase.

Reply:
Done.